README.md

```diff
@@ -5,20 +5,21 @@
 Runtime for isomorphic javascript libraries (that work in the browser and node.js environment) generated via [Autorest](https://github.com/Azure/Autorest).
 
 ## Requirements
-- node.js version > 6.x
-- npm install -g typescript
+- Node.js version > 6.x
+- `npm install -g typescript`
 
 ## Installation
 - After cloning the repo, execute `npm install`
 
 ## Execution
 
-### node.js
-- Set the subscriptionId and token
-- Run `node samples/node-sample.js`
+### Node.js
+- Set the subscriptionId and token as instructed in `samples/node-samples.ts`
+- Run `npx ts-node samples/node-sample.js`
```

Comment:
npx? I don't recall using this elsewhere in our projects. Any particular wins this gives us?

Reply:
`npx` is just an `npm` command to execute a dependency script without installing it globally. e.g. if you install tsc as a devdep in your project you can use `npx tsc` without having to `npm install -g tsc`

Reply:
yeah it's convenient for simple commands without needing to add an entry to the package.json script section.  It even installs the package if it's not available locally.